decidim-core/lib/decidim/component_notification_setting_manifest.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ComponentNotificationSettingManifest
+    include ActiveModel::Model
+    include Decidim::AttributeObject::Model
+
+    attribute :name, Symbol
+    attribute :area, Symbol
+    attribute :default_value, String, default: "1"
+
+    validates :default_value, inclusion: { in: %w(0 1) }
+    validates :area, inclusion: { in: [:global, :administrators] }
```

Comment:
Can we find other naming instead of `area`? I find it confusing as we already have the `Decidim::Area` model

Reply:
Also is the `:global` area being used? Or is for future proofing?

Reply:
```:global``` is not used yet, is for future proofing